includes/Stories_Renderer/Renderer.php

```diff
@@ -523,20 +550,14 @@ protected function render_story_with_poster() {
 		<?php
 		$this->get_content_overlay();
 
-		// Start collecting markup for the lightbox stories. This way we don't have to re-run the loop.
-		ob_start();
-
-		// Collect story links to fill-in non-AMP lightbox 'amp-story-player'.
-		if ( ! $this->is_amp_request() ) {
-			?>
-				<a href="<?php echo esc_url( $story_data->get_url() ); ?>"><?php echo esc_html( $story_data->get_title() ); ?></a>
-			<?php
-		} else {
-			// Generate lightbox html for the AMP page.
-			$this->generate_amp_lightbox_html();
+		if ( ! in_array( $story_data->get_id(), self::$stories_loaded, true ) ) {
+			if ( ! $this->is_amp_request() ) {
+				$this->generate_lightbox_html_noamp( $story_data );
+			} else {
+				$this->generate_amp_lightbox_html_amp( $story_data );
+			}
+			self::$stories_loaded[] = $story_data->get_id();
```

Comment:
Why does this need to be static? This feels like a normal classic class property to me.